docs/development.md

```diff
@@ -55,10 +55,11 @@ To run your plugin in development, you will need to configure a GitHub App to de
 1. [Create a new GitHub App](https://github.com/settings/apps/new) with:
     - **Webhook URL**: Set to `https://example.com/` and we'll update it in a minute.
     - **Webhook Secret:** `development`
-    - **Permissions & events** needed will depend on how you use the bot, but for development it may be easiest to enable everything.
-1. Download the private key and move it to the project directory
-1. Edit `.env` and set `APP_ID` to the ID of the app you just created.
-1. Run `$ npm start` to start the server, which will output `Listening on https://yourname.localtunnel.me`;
+    - **Permissions & events** can be found in the next tab over. What you need to choose will depend on what data you want the bot to have access to. Note: if, for example, you only enable issue events, you will not be able to listen on pull request webhooks with your bot. However, for development we recommend enabling everything.
```

Comment:
> …can be found in the next tab over…

This is on the same tab when creating a new integration.

> …want the **bot** to have access to…

I've started trying to avoid using the word "bot" as much as possible. I'm not totally opposed to using the word casually or when talking about Probot in the abstract, but for posterity, here's my thinking for wanting to avoid it in written documentation:

1. Calling it "a Probot bot" sounds weird, whereas "a Probot app" sounds better. So instead of using them interchangeably, I think it's clearer to use "app" everywhere.
2. The word "bot" is so trendy that I think it somewhat cheapens what Probot is. It might help it ride the wave in the near term, but it will eventually feel outdated.

Thoughts?